---
uint32 tracked_nodes
uint32 seen_nodes
uint32 alive_nodes
uint32 listed_nodes